### PR TITLE
cowqueue redo

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
@ryanchou-dev @ChuangSheep

File: solutions/bronze/usaco-713.mdx [hunk 1]
```diff
@@ -158,61 +159,121 @@ Since the time the cows arrive and need for questioning can be up to $10^6$, we
...
+		Arrays.sort(cows, Comparator.comparingInt(c -> c.arrival));
+
+		int currTime = 0;
+		for (Cow c : cows) {
+			// this cow was already waiting, add duration to the current time
+			if (currTime > c.arrival) {
+				currTime += c.duration;
+			} else {
+				// the last cow finished before this one arrived,
+				// so set the current time to when this cow finishes.
```
Comment (ChuangSheep) [inline solutions/bronze/usaco-713.mdx:237]: idk maybe multiline comment?

**Review by ChuangSheep — APPROVED** [review]
naisu ❤️

File: solutions/bronze/usaco-713.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/bronze/usaco-713.mdx:237]: with 2 lines, it doesn't matter too much
but if you insist, i'll convert to multiline

Comment (ChuangSheep) [reply solutions/bronze/usaco-713.mdx:237]: No it's alright

**Comment by ChuangSheep** [comment]
?

**Comment by SansPapyrus683** [comment]
lol thx for the reminder

**Pull request merged**